tests/forte/data/audio_annotation_test.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2022 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for AudioAnnotation.
+"""
+import os
+import unittest
+from typing import Dict
+from numpy import array_equal
+
+from forte.pipeline import Pipeline
+from forte.processors.base.pack_processor import PackProcessor
+from forte.data.data_pack import DataPack
+from forte.data.readers import AudioReader
+from forte.data.ontology.top import AudioAnnotation
+from ft.onto.base_ontology import Recording, AudioUtterance
+
+
+class RecordingProcessor(PackProcessor):
+    """
+    A processor to add a Recording ontology to the whole audio data.
+    """
+    def _process(self, input_pack: DataPack):
+        Recording(
+            pack=input_pack, begin=0, end=len(input_pack.audio)
+        )
+
```

Comment:
Let's add an example using the audio reader, the audio annotations and so on in the examples folder, to serve as tutorial materials.